app/views/organizations/staff/external_form_upload/_upload_results.html.erb

```diff
@@ -1,21 +1,21 @@
-<turbo-frame id="results">
```

Comment:
This was not actually doing anything with a turbo frame so a div is more appropriate.